app/models/miq_server/worker_management/monitor/systemd.rb

```diff
@@ -0,0 +1,73 @@
+module MiqServer::WorkerManagement::Monitor::Systemd
+  extend ActiveSupport::Concern
+
+  def cleaup_failed_systemd_services
```

Comment:
typo ^ cleanup

Reply:
LOL I was trying to figure out how this worked if I typo'd it up there, at least I was consistent